doc/api/esm.md

```diff
@@ -1130,18 +1130,41 @@ node --experimental-top-level-await b.mjs # works
 <!-- type=misc -->
 
 To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
-
+provided via `--experimental-loader ./loader-name.mjs` arguments to Node.js
 When hooks are used they only apply to ES module loading and not to any
 CommonJS modules loaded.
 
+Loaders are modules which export functions and data which can be used by
+Node.js to modify the behavior of loading ES Modules. Loaders may be "chained"
+together by passing multiple `--experimental-loader` arguments to Node.js. A
```

Comment:
Should we use a shorter `-l` flag to make this easier now?

Reply:
Short flags seem like something appropriate for non-experimental flags; experimental flagged things seem like something that shouldn't be made convenient.